test/check_configurable_requirements.py

```diff
@@ -333,7 +344,7 @@ def perform_review():
 
     if found:
         print(
-            f"Found {found} uses of versionable components not declared in get_requirements()"
+            f"Found {found} coding standards violations"
```

Comment:
I mean, technically accurate, but violation sounds a little bit strong?  Any chance we could tone it down a little please?  Don't want to put people off of contributing!  5:D

Reply:
For sure! Had a feeling that might be too much.